packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.tsx

```diff
@@ -73,7 +74,13 @@ export class GroupHeader extends BaseComponent<IGroupDividerProps, IGroupHeaderS
         })}
         style={viewport ? { minWidth: viewport.width } : {}}
         onClick={this._onHeaderClick}
+        onKeyDown={this._onKeyDown}
         aria-label={group.ariaLabel || group.name}
+        aria-selected={group.isSelected}
+        aria-expanded={!group.isCollapsed}
+        aria-level={(groupLevel || 0) + 1}
+        aria-posinset={(groupIndex || 0) + 1}
+        aria-setsize={group.children ? group.children.length : group.count}
```

Comment:
I would just go with group.count. group.children contains the sub-groups, if any, so a count of that is not an accurate reflection of number of items in that group. Also, we will possibly end up with something like ParentGroup (size=2), SubGroup1 (size = 10), SubGroup2 (size = 20) - that looks weird.